stable_diffusion/image2image.py

```diff
@@ -26,6 +26,7 @@
     args = parser.parse_args()
 
     sd = StableDiffusion()
+    sd.ensure_models_are_loaded()
```

Comment:
Just wondering, why did you add that? Is it to get better timing information?

Also just a thought, but maybe it would be simpler to have the models subclass `nn.Module` and then you can just do a regular `mx.eval` on the params / state.

Reply:
We need to fix some things in `nn.Module` I think cause deleting layers is not that straightforward anymore. Probably somewhere along the changes for state etc something made this not possible. So I 'll leave it for later.

Reply:
Sounds good to leave for later... but I didn't fully get your comment. How does `eval`ing the model params relate to deleting the layers?